### PR TITLE
Extra charts

### PR DESCRIPTION
**Pull request by xybrnet**
@longhotsummer

### CODE REVIEW

File: wazimap_za/profiles/youth.py [hunk 1]
```diff
@@ -20,6 +20,16 @@
...
+HOUSEHOLD_CROWDED_RECODE = {
+    'Yes': 'Overcrowded',
+    'No': 'Non-overcrowded'
+}
+
+COMPLETED_GRADE9_RECODE = {
+    'Yes': 'Completed grade 9 or higher',
+    'No': 'Have not completed grade 9 or higher'
+}
+
```
Comment (xybrnet) [inline wazimap_za/profiles/youth.py]: Should we do this, or rename the field keys in the tables as with the poverty indices?

Comment (longhotsummer) [inline wazimap_za/profiles/youth.py]: My vote is changing the raw data. The more I work with Wazimap, the more I think the raw data should be in the right form as far as possible.

**Comment by longhotsummer** [comment]
LGTM

File: wazimap_za/profiles/youth.py (hunk 1 above)
Comment (xybrnet) [inline wazimap_za/profiles/youth.py]: Cool, I also think it's better. I've kept the recoding for Grade 9 completion until it is confirmed that it should indeed be changed.

**Pull request merged**